source/quark/R/01_QuarkData.R

```diff
@@ -472,11 +472,62 @@ QuarkData$methods(
         data[ , targets] <<- tmp2
     },
 
-    cleanCollinVars = function(x)                                               {
-        "Remove one variable from all collinear pairs"
-        collinVars <<- x
-        removeVars(x = unique(collinVars$var1), reason = "collinear")
-    },
+    
+  cleanCollinVars = function(x)                                                {
+       "Remove one variable from all collinear pairs"
+        collinVars <- x
```

Comment:
You must use the deep assignment operator '<<-' to assign values to the fields of a reference class object. The normal assignment operator '<-' will only make a local assignment when employed within a reference class method (i.e., the underlying field is not modified). 

Please change line 478 to 'collinVars <<- x'